### ISSUE TITLE
BlenderBIM - Feature Request: update IfcOpeningElements when editing LayerThickness of IfcWallType

### ISSUE THREAD

**Issue by rvouilloz** [IfcOpenShell/IfcOpenShell#3157]
Would it be possible to automatically adjust the thickness of IfcOpeningElements when editing the LayerThickness value in an IfcWallType's IfcMaterialLayerSet? 
![1](https://github.com/IfcOpenShell/IfcOpenShell/assets/97536689/6b4283bc-fc0d-4c07-8cb8-07a0c5406f3f)

**Comment by Andrej730** [comment]
Any ideas of the cases when partial opening in the wall is needed and there is no need to adjust it?

**Comment by rvouilloz** [comment]
Yes, I would say that in the case of a splayed window it seems complicated to automate and preferable to leave as it is. At least if the IfcOpeningElement is a Tesselation, I don't know if the IfcOpeningElement supports parametric splay?
